app/containers/ITunesSearch/tests/index.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ *
+ * Tests for ITunesSearch
+ *
+ *
+ */
+
+import React from 'react';
+import { renderProvider, timeout } from '@utils/testUtils';
+import { fireEvent } from '@testing-library/dom';
+import { ITunesSearchTest as ITunesSearch } from '../index';
+
+describe('<ITunesSearch /> container tests', () => {
+  let submitSpy;
+
+  beforeEach(() => {
+    submitSpy = jest.fn();
+  });
+  it('should render and match the snapshot', () => {
+    const { baseElement } = renderProvider(<ITunesSearch dispatchGetSongName={submitSpy} />);
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should call dispatchGetSongName on change', async () => {
+    const { getByTestId } = renderProvider(<ITunesSearch dispatchGetSongName={submitSpy} />);
+    fireEvent.change(getByTestId('song-search-bar'), {
+      target: { value: 'song' }
+    });
+
+    await timeout(500);
+    expect(submitSpy).toBeCalled();
```

Comment:
dispatchGetSongName is invoked with `song` as the parameter

Reply:
correct. We just need to automate the test for it. 

so we can use 

```
const song = 'song'
 fireEvent.change(getByTestId('song-search-bar'), {
      target: { value: song }
    });

    await timeout(500);
expect(submitSpy).toBeCalledWith(song);